Code/Tools/AssetProcessor/native/ui/CacheServerData.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <native/utilities/AssetUtilEBusHelper.h>
+#include <native/utilities/PlatformConfiguration.h>
+#include <AzCore/IO/Path/Path.h>
+#endif
+
+namespace AssetProcessor
+{
+    struct CacheServerData
```

Comment:
Does this need to be a struct? Does everything need to be public?

Reply:
this is really to store POD plus some convenience functions. This originally was an internal struct that Joe wanted to be kicked out to its own code module, so it was meant to be internal but a previous code review wanted it to be changed.